test/packages/good_v3/manifest.yml

```diff
@@ -40,10 +40,22 @@ policy_templates:
         division: observability
         team: obs-infraobs-integrations
     configuration_links:
-      - title: Security overview
-        url: "/security/overview"
+      - title: View Agents
+        url: "/agents"
         type: next_step
-        content: "View security overview"
+        content: "Check your agents in Fleet"
+        appId: "fleet"
+        internal: true
```

Comment:
Or, trying to do this a bit more generic, define something like Android intents, but then we need to know the possible intents beforehand.
```
      - title: View Agents
        intent: "agents_view"
        type: next_step
        content: "Check your agents in Fleet"
```